### ISSUE TITLE
The connector cannot be run locally when following the README instructions.

### ISSUE THREAD

**Issue by ouphi** [eclipse-dataspaceconnector/DataSpaceConnector#189]
Following the [README instructions](https://github.com/eclipse-dataspaceconnector/DataSpaceConnector#run-your-first-connector) to run the connector does not work.

When running the following commands:

```bash
./gradlew :launchers:basic:shadowJar
java -jar launchers/basic/build/libs/dataspaceconnector-basic.jar
```

We get the following error:

```
SEVERE 2021-10-26T10:42:41.734764 Error booting runtime
org.eclipse.dataspaceconnector.spi.EdcException: Extension feature required by org.eclipse.dataspaceconnector.contract.ContractServiceExtension not found: edc:asset:assetindex
	at org.eclipse.dataspaceconnector.system.DefaultServiceExtensionContext.lambda$sortExtensions$8(DefaultServiceExtensionContext.java:168)
	at java.base/java.lang.Iterable.forEach(Iterable.java:75)
	at org.eclipse.dataspaceconnector.system.DefaultServiceExtensionContext.lambda$sortExtensions$9(DefaultServiceExtensionContext.java:165)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
	at org.eclipse.dataspaceconnector.system.DefaultServiceExtensionContext.sortExtensions(DefaultServiceExtensionContext.java:165)
	at org.eclipse.dataspaceconnector.system.DefaultServiceExtensionContext.loadServiceExtensions(DefaultServiceExtensionContext.java:136)
	at org.eclipse.dataspaceconnector.system.runtime.BaseRuntime.createExtensions(BaseRuntime.java:119)
	at org.eclipse.dataspaceconnector.system.runtime.BaseRuntime.boot(BaseRuntime.java:62)
	at org.eclipse.dataspaceconnector.system.runtime.BaseRuntime.main(BaseRuntime.java:41)
```

**Comment by paullatzelsperger** [comment]
We do need workings samples ASAP, I'll open another Issue that fixes that and also adds sanity checks that build and run the samples on the CI pipeline.

**Comment by jimmarino** [comment]
@paullatzelsperger has this been resolved?

**Comment by paullatzelsperger** [comment]
I think this was fixed by #201
